Add FAQ & Troubleshooting section to Destination Actions docs (#4193)
segmentio/segment-docs: src/connections/destinations/actions.md

> - You need to filter data from multiple types of call (for example, Track, Page, and Identify calls)
>
> If your use case does not match these criteria, you might benefit from using Mapping-level Triggers to match only certain events.

## FAQ & Troubleshooting

### Validation error when using the Event Tester

A validation error with the description "You may not have any subscriptions that match this event" will surface when sending an event with an Actions destination's Event Tester that does not match the trigger of any configured and enabled mappings. To resolve this, either a mapping must be created with a trigger to handle the event being tested, or the test event's payload must be updated to match the trigger of any existing mappings. 

### Multiple mappings triggered by the same event

> A request will be generated for each mapping that's configured to trigger on an event. For example, with an event named "Subscription Updated", if two mappings are enabled and both have conditions defined to trigger on "Subscription Updated" events, two requests will be generated and sent to the destination for each "Subscription Updated" event. 

Reviewer: ```suggestion
When the same event triggers multiple mappings, a request will be generated for each mapping that's configured to trigger on an event. For example, for the *Subscription Updated* event, if two mappings are enabled and both have conditions defined to trigger on the *Subscription Updated* event, the two requests will be generated and sent to the destination for each *Subscription Updated* event. 
```
